src/main/java/mate/academy/mapper/BookMapper.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.mapper;
+
+import mate.academy.config.MapperConfig;
+import mate.academy.dto.BookDto;
+import mate.academy.dto.CreateBookRequestDto;
+import mate.academy.model.Book;
+import org.mapstruct.Mapper;
+
+@Mapper(config = MapperConfig.class)
+public interface BookMapper {
+
+    BookDto toDto(Book book);

```

Comment:
Consider adding null checks or handling for the toDto method to avoid potential NullPointerExceptions when the input Book object is null.

Reply:
Is it not enough, when the checking is automatically generated by MapStruct ?